### ISSUE TITLE
Add documentation about `alias_tool` to User Guide

### ISSUE THREAD

**Issue by AlexWayfer** [dazuma/toys#193]
I don't see any deprecation warning.

And I see only [documentation about `:delegate_to`](https://dazuma.github.io/toys/gems/toys/v0.14.1/file.guide.html#delegating-tools), but not about `alias_tool`, although their behavior is different due to [`alias_tool` documentation](https://dazuma.github.io/toys/gems/toys/v0.14.1/Toys/DSL/Tool.html#alias_tool-instance_method).

**Comment by dazuma** [comment]
Yeah I don't have any plans to deprecate `alias_tool`. Not every feature is described in detail in the user's guide, but you're right, it's probably a good idea to mention that `alias_tool` exists.
